pkg/server/event/sync/convert.go

```diff
@@ -42,8 +42,12 @@ import (
 func (c *CR2UX) ConvertApp2DatastoreApp(ctx context.Context, targetApp *v1beta1.Application) (*DataStoreApp, error) {
 	cli := c.cli
 	appName := c.getAppMetaName(ctx, targetApp.Name, targetApp.Namespace)
+	projectName := model.DefaultInitName
+	if labels := targetApp.Labels; labels != nil && labels[model.LabelSyncProject] != "" {
```

Comment:
There are three rules to decide the project:
1. If the application is an addon, the project name is `system`;
2. If the application's namespace is bound to an environment, the project name is the same as this environment.
3. For Others, the default project name is `default`.

Now, we add a label to specify the project name, So what are his priorities? This needs to think.

Reply:
* The addon should have the first priority, the second priority should be the marked project.
* Shall we use annotation instead of label?